webpack-helper.js

```diff
@@ -22,14 +22,17 @@ function ensureLoader(loaders, test, loader) {
  * @param  {Object}  config Base webpack config
  * @param  {Boolean} dev    Whether to render in Dev mode (debugger, improved errors, etc)
  * @param  {Boolean} test   Whether to render in Test mode (additional exposed functionality for tests)
+ * @param  {String} modulesProp the webpack config prop for 'modules' (determined
+ *                              by whether to use the webpack 1.x config style)
  *
  * @return {Object}         Updated webpack config object
  */
-module.exports = function(config) {
+module.exports = function(config, dev, test, modulesProp) {
+
   config.resolveLoader = config.resolveLoader || {};
-  config.resolveLoader.modules = config.resolveLoader.modules || [];
+  config.resolveLoader[modulesProp] = config.resolveLoader[modulesProp] || [];
```

Comment:
since this function is feasibly accessed directly, might be good to default modulesProp if it isn't set